scripts/ci/travis-install.sh

```diff
@@ -35,7 +35,7 @@ else
   ./scripts/build/check-dependency.sh make
   ./scripts/build/check-dependency.sh npm
 
-  npm install -g npm@3.10.10
+  npm install -g npm@3.10.10 uglify-es@3.0.3
```

Comment:
So, there seems to be a number of packages which we're now installing into each of Docker, Travis and Appveyor. To reduce duplication (and to ensure that all the version-numbers stay in sync), perhaps we should add some kind of meta-file(s) that all three processes read? (And perhaps that/those same meta-file(s) would also make the Etcher development-environment easier to set up?)